packages/core/src/Selection.ts

```diff
@@ -28,7 +28,7 @@ export function isSelection(x: any): x is Selection {
   return x instanceof Selection;
 }
 
-function create(options: SelectionResolverOptions, include?: Selection | Selection[]): Selection {
+function create(options: SelectionResolverOptions, include?: Selection[]): Selection {
```

Comment:
Why the change in types here? The code was designed to accept either a single selection or an array. And if we make the change, the code below could be adjusted to avoid the array wrap and `flat` call.

Reply:
Ah, I misinterpreted the constructor and logic here. I changed this to `include ? [include].flat() : undefined` so that the code typechecks correctly.